src/main/java/com/knowledgepixels/registry/MainVerticle.java

```diff
@@ -57,6 +59,16 @@ public void start(Promise<Void> startPromise) throws Exception {
 		router.route(HttpMethod.GET, "/style.css").handler(c -> {
 			ResourcePage.show(c, "style.css", "text/css");
 		});
+
+		// Metrics
+		final var metricsHttpServer = vertx.createHttpServer();
+		final var metricsRouter = Router.router(vertx);
+		metricsHttpServer.requestHandler(metricsRouter).listen(9293);
```

Comment:
Please update docker-compose.yml to expose this to localhost only, along with a comment that it's for the metrics. This should make it easier for people to get started with the endpoint.